### ISSUE TITLE
godot 4.2.2 crashes on android when I decide to change the modulate

### ISSUE THREAD

**Issue by LMisters** [godotengine/godot#91784]
### Tested versions

...,4.2.1 - 4.2.2

### System information

Android 9 Chipset	Exynos 8895 (10 nm) - EMEA Qualcomm MSM8998 Snapdragon 835 (10 nm) - USA & China CPU	Octa-core (4x2.3 GHz Mongoose M2 & 4x1.7 GHz Cortex-A53) - EMEA Octa-core (4x2.35 GHz Kryo & 4x1.9 GHz Kryo) - USA & China GPU	Mali-G71 MP20 - EMEA Adreno 540 - USA & China

### Issue description

https://github.com/godotengine/godot/assets/169423403/f3b4c003-de11-4494-b608-aeddbccc3e8b

### Steps to reproduce

https://github.com/godotengine/godot/assets/169423403/9be9f57b-7221-41bf-97d7-5ad3d99365f0

### Minimal reproduction project (MRP)

Is a bug

**Comment by AThousandShips** [comment]
Please describe the steps in words as well as a video, it's very hard to follow along to this video

**Comment by AThousandShips** [comment]
Closing due to lack of response, if you provide the requested information this can be reopened
